packages/slate-editor/src/extensions/flash-nodes/components/FlashNodes.tsx

```diff
@@ -1,12 +1,12 @@
+import type { SlateEditor, TNode } from '@udecode/plate-common';
+import { toDOMNode, useEditorState } from '@udecode/plate-common/react';
 import type { RefObject } from 'react';
 import React, { useEffect, useState } from 'react';
-import type { Editor, Node } from 'slate';
-import { ReactEditor, useSlateStatic } from 'slate-react';
 
 import styles from './FlashNodes.module.scss';
 
 export function FlashNodes({ containerRef }: { containerRef: RefObject<HTMLDivElement> }) {
-    const editor = useSlateStatic();
+    const editor = useEditorState();
```

Comment:
`useSlateStatic()` has been replaced with `useEditorState()`. Is is not a mistake?

Plate's alternative `useSlateStatic()` is `useEditorRef()`, right?

Reply:
Nevermind. I see it was intentional: 3d755b7530e9a43e9dff75f1188530a6c97cdfd7

Reply:
Yeah, with `useEditorRef` the component would not re-render if `editor.nodesToFlash` would change.